using-mobile-extensions/adobe-campaignclassic/README.md

```diff
@@ -72,9 +72,9 @@ Time, in seconds, to wait for a response from the registration or tracking endpo
    You can add the following pods to your `Podfile`:
 
    ```text
-   pod 'ACPCampaignClassic', '2.0.0'
-   pod 'ACPLifecycle', '2.0.0'
-   pod 'ACPCore', '2.0.0'
+   pod 'ACPCampaignClassic', '~> 2.0' 
+   pod 'ACPLifecycle', '~> 2.0' 
+   pod 'ACPCore', '~> 2.0'
```

Comment:
Let's omit the versions entirely. I don't think we are specifying versions in the setup elsewhere.

Reply:
let's use dynamic version range 1.+ for android as well on L58.